data/favicon.json

```diff
@@ -406,6 +406,7 @@
 	"tidbit5150.com": "tidbit5150.com.png",
 	"timbusinessmail.it": "timbusiness.it.ico",
 	"tk.de": "tk.de.ico",
+	"tracker-software.com": "tracker-software.com.png",
```

Comment:
There did you get the icon from? If I visited the site today it did show a different favicon (https://trackerwebsitestatic.cachefly.net/cdn/bundles/app/images/favicon.ico)

Reply:
Actually, I had it locally stored (for use with another addon). I just recently saw, that their mails use dkim signatures, but didn't look at their homepage, which got a redesign.
https://web.archive.org/web/20140308061603/http://www.tracker-software.com/
I've updated the icon to the latest one.